news/fixed_latex.rst

```diff
@@ -0,0 +1,12 @@
+**Added:** None
+
+**Changed:**
+ * fixed latex preamble syntax error due to matplotlib update

```

Comment:
should be a fix not a change.  It is really a bug fix, not a change in behavior.